README.md

```diff
@@ -81,6 +91,41 @@ $ rpc-client/build/install/rpc-client/bin/rpc-client issue -u localhost:10011 -n
 
 The client will discover all the participants in the network and prompt you for a node as the counterprise (borrower), make sure to pick the other node you created in the network, not the node you are connected to or the notary.
 
+### Account Utilities
+
```

Comment:
can you add a short description of what accounts are, and how are they different from the node's legal identity?

Reply:
Sure. Accounts are very different from node's legal identity. Will update the docs.